torch/onnx/utils.py

```diff
@@ -1844,6 +1845,24 @@ def _run_symbolic_function(
         env=env,
     )
 
+    # when domain is "aten" and operator_export_type = ONNX_ATEN or ONNX_ATEN_FALLBACK
+    # create org.pytorch.aten::ATen oeprator regardless of symbolics exist or not
+    if _should_aten_fallback(ns_op_name, opset_version, operator_export_type):
```

Comment:
I can see we want the behavior for `ONNX_ATEN`. Do we want this behavior for `ONNX_ATEN_FALLBACK` as well? Should it try to translate first before falling back to Aten?

Reply:
outdated code/comment. `ONNX_ATEN_FALLBACK` is not handled here nor changed by this PR